### ISSUE TITLE
Order of delivery for ORD tables

### ISSUE THREAD

**Issue by hchagani-oasislmf** [OasisLMF/OpenDataStandards#29]
Following discussions with @MattDonovan82:

- [x] EPT (Exceedance Probability Table) - ktools `v3.6.0` PR: https://github.com/OasisLMF/ktools/pull/195
- [x] PSEPT (Per Sample Exceedance Probability Table) - ktools `v3.6.0` PR: https://github.com/OasisLMF/ktools/pull/195
- [x] SELT (Sample Event Loss Table) - ktools `v3.6.1` PR: https://github.com/OasisLMF/ktools/pull/204
- [x] PALT (Period Average Loss Table) - ktools `v3.6.2` PR: https://github.com/OasisLMF/ktools/pull/218
- [x] MELT (Moment Event Loss Table) - ktools `v3.6.4` PR: https://github.com/OasisLMF/ktools/pull/229
- [x] MPLT (Moment Period Loss Table) - ktools `v3.6.4` PR: https://github.com/OasisLMF/ktools/pull/229
- [x] SPLT (Sample Period Loss Table) - ktools `v3.6.4` PR: https://github.com/OasisLMF/ktools/pull/229
- [x] QELT (Quantile Event Loss Table) - ktools `v3.7.0` PR: https://github.com/OasisLMF/ktools/pull/238
- [x] QPLT (Quantile Period Loss Table) - ktools `v3.7.0` PR: https://github.com/OasisLMF/ktools/pull/238

#### Outstanding Issues
- [ ] SELT missing `ImpactedNumLocs` field, issue: https://github.com/OasisLMF/ktools/issues/207
- [ ] PALT missing `SourceNumLocs` and `SourceExposure` fields, issue: https://github.com/OasisLMF/ktools/issues/221
- [ ] MELT missing fields, issue https://github.com/OasisLMF/ktools/issues/232
- [ ] MPLT missing fields, issue https://github.com/OasisLMF/ktools/issues/233
- [ ] SPLT missing fields, issue https://github.com/OasisLMF/ktools/issues/234

**Comment by RajuSharmaFaraday** [comment]
Hi,
Can anyone share the SQL scripts for creating the Database and tables for OED. I could not find it anywhere.

Thanks
Raj

**Comment by MattDonovan82** [comment]
please send me your email address and I'll send them over directly. 

Matt

**Comment by RajuSharmaFaraday** [comment]
raju.sharma@faraday.com

Thanks

**Comment by MattDonovan82** [comment]
scripts emailed.

**Comment by benhayes21** [comment]
Can we add some ORD documentation, please?

**Comment by benhayes21** [comment]
https://github.com/OasisLMF/OpenDataStandards/tree/feature/ord-worked-examples/OpenResultsData/Docs/WorkedExamples

**Comment by johcarter** [comment]
Implementation details for some ORD fields @hchagani-oasislmf 

SELT/SPLT
**ImpactedExposure**  - in gulcalc for each sample and item there is either a loss or no loss.  If there is a loss then the ImpactedExposure is the TIV for the related coverage. If loss = 0 then ImpactedExposure = 0.
For ImpactedExposure by summary_id, sum ImpactedExposure by sample across the items related to the summary_id.

MELT/MPLT
**MaxImpactedExposure** - the maximum of Impacted Exposure across the samples in the SELT/SPLT
**MeanImpactedExposure** - the mean of Impacted Exposure across the samples in the SELT/SPLT

MELT/MPLT
**MaxLoss** - calculated from the underlying effective damage distribution. It is the deterministic maximum possible loss. From getmodel output take the first damage bin which has cumulative probability = 1 (meaning there is no chance of sampling a higher damage bin) and then multiply the bin_to value (upper damage threshold) of that damage bin by the related coverage TIV.

**Comment by MattDonovan82** [comment]
so can this now be closed? @sambles

**Comment by benhayes21** [comment]
closing down as not an ODS issue
